src/main/kotlin/org/marissa/modules/MiscUtils.kt

```diff
@@ -0,0 +1,38 @@
+package org.marissa.modules
+
+import org.marissa.lib.Response
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+fun tellTheTime(trigger: String, response: Response) {
+    response.send(LocalDateTime.now().format(DateTimeFormatter.ofPattern("EEE dd MMM yyyy -- HH:mm.ss")))
```

Comment:
we should also change this to be UK time, or pass in the time. Marissa time currently is EST or something